go/sqldb/conn_params.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package sqldb defines an interface for low level db connection
+package sqldb
+
+// ConnectionParams contains all the parameters to use to connect to mysql
+type ConnectionParams struct {
```

Comment:
Since the connection is "type Conn", maybe we should rename this to "ConnParams"?